components/bl-reCaptcha-component/README.md

```diff
@@ -0,0 +1,59 @@
+# ReCaptcha
+
+ComponentName is a component of Backendless UI-Builder designer. This allows you to upgrade your security from bots when need to send some form
+
+In this component we use [ReCaptcha v2](https://www.google.com/recaptcha/about/).
+
+## Properties
+
+| Property | Type                                             | Default Value  | Logic       | Data Binding | UI Setting | Description                                                     |
+|----------|--------------------------------------------------|----------------|-------------|--------------|------------|-----------------------------------------------------------------|
+| Type     | Select ["image", "audio"]                        | "image"        |             | NO           | YES        | Allows to determine the type of verification by image or audio. |
+| Size     | Select ["compact", "normal", "invisible"]        | "normal"       | Size Logic  | YES          | YES        | Allows to determine the reCaptcha size.                         |
+| Badge    | Select ["bottom-right", "bottom-left", "inline"] | "bottom-right" |             | NO           | YES        | Allows to determine the position. Work when Size is inline.     |
+| Theme    | Select ["light", "dark"]                         | "light"        | Theme Color | YES          | YES        | Allows to determine the theme.                                  |
+
+## Action
+
+| Action        | Inputs | Return                                         |
+|---------------|--------|------------------------------------------------|
+| Get Token     |        | `String`: can get when the user passed captcha |
+| Get Is Passed |        | `Boolean`: `true` when the user passed captcha |
+
```

Comment:
I think Settings should be here
[example](https://github.com/Backendless/ui-builder-library/tree/main/components/bl-mapbox-component#settings)